recipe/build.sh

```diff
@@ -15,6 +15,12 @@ patch arithchk.c ${RECIPE_DIR}/patch_arithchk
 # but adding the -fPIC option to the CFLAGS
 sed 's/CFLAGS = -O/CFLAGS = -O -fPIC/g' makefile.u > Makefile
 
+# The Makefile directly calls "cc", which is sometimes not the name
+# of the compiler. So we create a link here and add this directory
+# to PATH
+ln -s ${CC} cc
+export PATH=${PATH}:${PWD}
```

Comment:
Does it hard-code the path to `cc` anywhere?

Reply:
No, this just lives as long as the build process. We are building in a temp directory, and only the things that we copy back under ${PREFIX} will become part of the package. As long as we don't copy that link (and we don't), we are fine.

Reply:
Right, was more asking if `f2c` might hard-code the compiler location within the binary somehow.